docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,131 @@
 
--   Limit the total counts in each panel to a specific set of severities, repositories, or security categories by clicking the filter drop-downs above the main area.
+To limit the information displayed in each panel to a specific set of repositories, use the filter drop-down above the main area.
 
--   Review the [severity assignment rules](#item-severities-and-deadlines) by clicking the **See rules** button in the top right-hand corner of the page.
+### Open findings overview
 
-![Security and risk management dashboard](images/security-risk-management-dashboard.png)
+The **Open findings overview** panel displays the total number of open findings, as well as the number of findings of each severity.
 
-## Item list
+To access the findings page with the corresponding filter applied, click on a number.
 
-The **Security and risk management items** page displays a filtered list of items, sorted by due date ascending.
+![Security and risk management open findings overview panel](images/security-risk-management-overview-open.png)<!-- TODO TAROT-2591: Screenshot -->
 
-To access the item list, access the [dashboard](#dashboard) and click the **Review** button in the area of interest, based on the desired filtering.
+### Open finding distribution
 
-When viewing the item list, you can:
+The **Open finding distribution** panel displays the relative distribution of open findings by scan kind, severity, or status.
 
--   Update the filtering criteria by clicking the **Severity**, **Status**, **Repository**, or **Security category** drop-downs above the list.
+To select the desired distribution, use the drop-down in the top right-hand corner of the panel.
 
--   Find out more about an item by clicking its **Details** column to navigate to the item of interest on the source platform.
+To access the findings page with the corresponding filter applied, click on a number.
 
--   Review the [severity assignment rules](#item-severities-and-deadlines) by clicking the **See rules** button in the top right-hand corner of the page.
+![Security and risk management open finding distribution panel](images/security-risk-management-overview-distribution.png)<!-- TODO TAROT-2591: Screenshot -->
 
-![Security and risk management items](images/security-risk-management-item-list.png)
+### Open finding history
 
-## Exporting the security item list
+The **Open finding history** panel displays a graph showing weekly trends of open findings from the past three months, grouped by severity.
+
+For a detailed view of the distribution on a specific week, hover over the graph.
+
+![Security and risk management open finding history panel](images/security-risk-management-overview-history-open.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Activity history
+
+The **Activity history** panel displays a graph showing weekly counts of open and closed findings over the past three months and the overall open finding trend.
```

Comment:
findings here again (?) 😅

Reply:
Yup, the idea is to mention that we have weekly counts (open/closed) plus the overall trend. I'll rephrase.